lib/api/src/grpc/proto/points_service.proto

```diff
@@ -79,6 +79,10 @@ service Points {
   */
   rpc RecommendGroups (RecommendPointGroups) returns (RecommendGroupsResponse) {}
   /*
+  Compute the average vector from a list of point IDs or vectors
+  */
+  rpc AverageVector (AverageVectorPoints) returns (AverageVectorResponse) {}
```

Comment:
normally, we would prefer to implement new type of scoring strategy as a part of `query` endpoint, instead of introducing a new one